.github/workflows/windows-qt6.yml

```diff
@@ -0,0 +1,109 @@
+---
+name: 🪟 Windows Qt6
+on:
+  push:
+    branches:
+      - master
+      - release-**
+  pull_request:
+  release:
+    types: ['published']
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: build (windows)
+    runs-on: windows-2022
+
+    steps:
+      - name: 🐣 Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: 🐩 Install CMake and Ninja
+        uses: lukka/get-cmake@latest
+        with:
+          cmakeVersion: 3.29.2
+
+      - name: 🧽 Developer Command Prompt for Microsoft Visual C++
+        uses: ilammy/msvc-dev-cmd@v1
+        with:
+          # See https://github.com/actions/runner-images/issues/9398
+          toolset: 14.39
+
+      - name: Setup vcpkg
+        uses: ./.github/actions/setup-vcpkg
+
+      - name: Setup flex/bison
+        uses: robinraju/release-downloader@v1.10
+        with:
+          repository: 'lexxmark/winflexbison'
+          fileName: '*.zip'
+          tag: 'v2.5.24'
```

Comment:
```suggestion
          # v2.5.25 is broken and will lead to random build failures because of race conditions, see https://github.com/lexxmark/winflexbison/issues/86
          tag: 'v2.5.24'
```